### ISSUE TITLE
[BUG] Metrics don't display for version 1.8 or later on port!=5000

### ISSUE THREAD

**Issue by LBerth** [mlflow/mlflow#3475]
Hi everyone,

### Willingness to contribute
- [X] No. I cannot contribute a bug fix at this time.

### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**: Yes, custom code
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: CentOS Linux 7 (Core)
- **MLflow installed from (source or binary)**: pip
- **MLflow version (run ``mlflow --version``)**: 1.8 or later
- **Python version**: 3.7
- **npm version, if running the dev UI**: --
- **Exact command to reproduce**: mlflow ui --port=5001

### Describe the problem
As we are several users working on the same machine, we often use different ports for the server so that each user can display his own mlflow tracking UI.
But after updating mlflow from version 1.7.2 to 1.8 or later, we experienced a bug in the metric display interface : the line graphs are not displayed, there is only a blank pane instead. This bug appears only when launching the server on ports different than the usual 5000.

Metric display for mlflow version 1.7.2 on every port:
![CaptureMlflow172](https://user-images.githubusercontent.com/19271957/94706698-d323dd80-0342-11eb-815f-5fbb74508d2b.PNG)

Metric doesn't display for mlflow version 1.8 or later when port != 5000:
![CaptureMlflow19-2](https://user-images.githubusercontent.com/19271957/94706775-eafb6180-0342-11eb-9591-5e96fca91853.PNG)

However when there is only one value for a metric, the bar plot is correctly displayed.
Other than that, the interface works fine.

### Code to reproduce issue
```
import mlflow

with mlflow.start_run(run_name='Test'):
    for i in range(10):
        mlflow.log_metric('metric',i, step=i)
```

### Other info / logs
Our web navigator is Firefox v79.0. We work behind the proxy of our company.

There isn't any error displayed in the mlflow terminal when this bug happens.

### What component(s), interfaces, languages, and integrations does this bug affect?
Components 
- [X] `area/tracking`: Tracking Service, tracking client APIs, autologging

Thank you for your help,
Lea

**Comment by harupy** [comment]
@LBerth Thanks for reporting this. Can you check if an error appears in the developer console on Firefox?

**Comment by LBerth** [comment]
Yes, there are 2 error messages and many warnings, here is the screenshot of the console:
![mlflow-web-console](https://user-images.githubusercontent.com/19271957/95199611-60e94800-07dd-11eb-9229-64786945c9ce.png)
